examples/multimodal/layer_specs.py

```diff
@@ -122,12 +124,16 @@ def get_layer_spec_te(is_vit=False, padding=False) -> ModuleSpec:
         ),
     )
 
+
 def get_mamba_layer_spec_te(padding=False) -> ModuleSpec:
     attn_mask_type = AttnMaskType.causal
     # Padding mask is needed for e.g. Context Parallel.
     if padding:
         attn_mask_type = AttnMaskType.padding_causal
 
+    assert TELayerNormColumnParallelLinear is not None
```

Comment:
Creating a not_none typing utility (or just using the one in PyTorch) might make your lives a bit easier.